src/Sentry/Protocol/Transaction.cs

```diff
@@ -55,14 +73,14 @@ public string Operation
         public SpanStatus? Status
         {
             get => Contexts.Trace.Status;
-            private set => Contexts.Trace.Status = value;
+            set => Contexts.Trace.Status = value;
         }
 
         /// <inheritdoc />
         public bool IsSampled
         {
             get => Contexts.Trace.IsSampled;
-            set => Contexts.Trace.IsSampled = value;
+            internal set => Contexts.Trace.IsSampled = value;
```

Comment:
`IsSampled` is nullable in Java

https://github.com/getsentry/sentry-java/blob/3c87a48442e536a41ed905d9e43bdc2f7b800176/sentry/src/main/java/io/sentry/SentryTraceHeader.java#L58